templates/pwa/serviceworker_js.tmpl

```diff
@@ -0,0 +1,70 @@
+var STATIC_CACHE = 'static-cache-v1';
+var urlsToCache = [
+  // js
+  '{{AppSubUrl}}/vendor/plugins/jquery.areyousure/jquery.are-you-sure.js',
+  '{{AppSubUrl}}/vendor/plugins/jquery/jquery.min.js',
+  '{{AppSubUrl}}/vendor/plugins/semantic/semantic.min.js',
+  '{{AppSubUrl}}/js/index.js?v={{MD5 AppVer}}',
+  '{{AppSubUrl}}/js/draw.js',
+  '{{AppSubUrl}}/vendor/plugins/clipboard/clipboard.min.js',
+  '{{AppSubUrl}}/vendor/plugins/gitgraph/gitgraph.js',
+  '{{AppSubUrl}}/vendor/plugins/vue/vue.min.js',
+  '{{AppSubUrl}}/vendor/plugins/emojify/emojify.min.js',
+  '{{AppSubUrl}}/vendor/plugins/cssrelpreload/loadCSS.min.js',
+  '{{AppSubUrl}}/vendor/plugins/cssrelpreload/cssrelpreload.min.js',
+  '{{AppSubUrl}}/vendor/plugins/dropzone/dropzone.js',
+  '{{AppSubUrl}}/vendor/plugins/highlight/highlight.pack.js',
+  '{{AppSubUrl}}/vendor/plugins/jquery.datetimepicker/jquery.datetimepicker.js',
+  '{{AppSubUrl}}/vendor/plugins/jquery.minicolors/jquery.minicolors.min.js',
+  '{{AppSubUrl}}/vendor/plugins/codemirror/addon/mode/loadmode.js',
+  '{{AppSubUrl}}/vendor/plugins/codemirror/mode/meta.js',
+  '{{AppSubUrl}}/vendor/plugins/simplemde/simplemde.min.js',
+
+  // css
+  '{{AppSubUrl}}/vendor/assets/font-awesome/css/font-awesome.min.css',
+  '{{AppSubUrl}}/vendor/assets/octicons/octicons.min.css',
+  '{{AppSubUrl}}/vendor/plugins/simplemde/simplemde.min.css',
+  '{{AppSubUrl}}/vendor/plugins/gitgraph/gitgraph.css',
+  '{{AppSubUrl}}/vendor/plugins/tribute/tribute.css',
+  '{{AppSubUrl}}/vendor/plugins/semantic/semantic.min.css',
+  '{{AppSubUrl}}/css/index.css?v={{MD5 AppVer}}',
+  '{{AppSubUrl}}/vendor/plugins/highlight/github.css',
+  '{{AppSubUrl}}/vendor/plugins/jquery.minicolors/jquery.minicolors.css',
+  '{{AppSubUrl}}/vendor/plugins/jquery.datetimepicker/jquery.datetimepicker.css',
+  '{{AppSubUrl}}/vendor/plugins/dropzone/dropzone.css',
+  '/css/theme-{{DefaultTheme}}.css',
```

Comment:
```
{{if ne DefaultTheme "gitea"}}
 	'{{AppSubUrl}}/css/theme-{{DefaultTheme}}.css',
{{end}}
```

cc: @lunny

Reply:
Of course feel free to extend the list at any time :smile::+1:

Reply:
@SohnyBohny thanks. I've added that in. @lunny the error you have shouldn't happen anymore, please try again.